### ISSUE TITLE
total_sensitivity_index in Sobol_G.py 

### ISSUE THREAD

**Issue by jyangfsu** [SALib/SALib#461]
The total_sensitivity_index in Sobol_G is likely not correct.  

```
sum_pv = pv.sum(axis=0)
# should be 
product_pv = np.product(1 + pv, axis=0)
```
 
```
return np.subtract(1, np.divide(np.subtract(sum_pv, pv.T), tv)) 
# should be 
return np.divide(pv * np.divide(product_pv, 1 + pv.T),  tv)
```

**Comment by ConnectedSystems** [comment]
@jyangfsu Thanks for raising this issue.

@jdherman I've checked the paper and the suggested change seems to match. The implementation in [`gsa_module`](https://github.com/damar-wicaksono/gsa-module/blob/develop/gsa_module/test_functions/sobol_g_curved.py) also seems to line up with the suggested change as well

I am running some comparisons now to confirm.

pinging @QianWanghhu as this is likely related to #335

**Comment by ConnectedSystems** [comment]
Confirmed that the new results line up with the analytic values provided by @QianWanghhu, will submit a PR

<html>
<body>


Original | Suggested in #461 | Results provided in Issue #335
-- | -- | --
0.789828 | 0.787144 | 0.787144
0.252684 | 0.242198 | 0.242198
0.097311 | 0.034317 | 0.034317
0.080798 | 0.010460 | 0.010460
0.073707 | 0.000105 | 0.000105
0.073707 | 0.000105 | 0.000105
0.073707 | 0.000105 | 0.000105
0.073707 | 0.000105 | 0.000105


</body>
</html>

**Comment by ConnectedSystems** [comment]
Closed by #464
